build/Dockerfile

```diff
@@ -112,11 +111,15 @@ ENV LIBRARY_PATH="$CUDA_HOME/lib64/stubs"
 RUN dnf install -y python3.11 git \
     && ln -s /usr/bin/python3.11 /bin/python \
     && python -m ensurepip --upgrade \
+    && dnf update -y \
     && dnf clean all
 
 # Removes the example private key to avoid high severity vulnerability warning
 RUN rm -f /usr/share/doc/perl-Net-SSLeay/examples/server_key.pem
 
+# Removes the python3.9 code to eliminate possible CVEs.  Also removes dnf
+RUN rpm -e $(dnf repoquery python3-* -q --installed) dnf python3 yum crypto-policies-scripts
```

Comment:
this uninstalls just python3.9 and not all of python3 correct?

Reply:
interesting that python --> python3.11 but python3 --> python3.9

Reply:
Right, just checked, 
```
python --version
Python 3.11.7

python3.11 --version
Python 3.11.7
```